resources/ext/ezbake.conf

```diff
@@ -2,71 +2,21 @@ ezbake: {
    pe: {}
    foss: {
       redhat: { dependencies: ["puppet-agent"],
-               # TODO: With AIO packages, ruby-load-path is now the same
-               # across all platforms, so we don't need this. It should be
-               # moved to a regular config file. See SERVER-331.
-               install: ["echo \\\"os-settings: {\\\"                         > %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf",
-                         "echo \\\"    ruby-load-path: [/opt/puppetlabs/agent/lib/ruby/vendor_ruby]\\\" >> %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf" ],
-               # This is terrible, but we need write access to puppet's
-               # var/conf dirs, so we need to add ourselves to the group.
-               # Then we need to chmod some dirs until the Puppet packaging
-               # is changed to allow group write.
-               # TODO: The Puppet agent package is currently missing some
-               # things. We should clean this up if possible once puppet-agent
-               # packages are settled. See SERVER-333.
-               postinst: ["install --directory --group=puppet --owner=puppet /var/lib/puppet/jruby-gems",
-                          "usermod -d /opt/puppetlabs/agent/cache puppet",
-                          "chown puppet:puppet /opt/puppetlabs/agent/cache",
-                          "chmod 775 /opt/puppetlabs/agent/cache",
-                          "install --directory /opt/puppetlabs/agent/cache/reports",
-                          "chown puppet:puppet /opt/puppetlabs/agent/cache/reports",
-                          "chmod 775 /opt/puppetlabs/agent/cache/reports",
-                          "echo \\\"[master]\\\"                               > /etc/puppetlabs/agent/puppet.conf"
-                          "echo \\\"vardir = /opt/puppetlabs/agent/cache\\\"   >> /etc/puppetlabs/agent/puppet.conf",
-                          "echo \\\"logdir = /var/log/puppetlabs\\\"           >> /etc/puppetlabs/agent/puppet.conf",
-                          "echo \\\"rundir = /var/run/puppetlabs\\\"           >> /etc/puppetlabs/agent/puppet.conf",
-                          "chown puppet:puppet /etc/puppetlabs/agent/",
-                          "chmod 775 /etc/puppetlabs/agent",
-                          "install --directory /etc/puppetlabs/agent/ssl",
-                          "chown -R puppet:puppet /etc/puppetlabs/agent/ssl",
-                          "chmod -R 770 /etc/puppetlabs/agent/ssl",
-                          "chown -R puppet:puppet /var/log/puppetlabs/",
-                          "chmod -R 770 /var/log/puppetlabs"]
+               postinst: [
+                 "/opt/puppetlabs/puppet/bin/puppet config set --section master vardir  /opt/puppetlabs/server/data/puppetserver",
```

Comment:
Ok, I don't have a big problem with it.  Was just curious if it was needed.  Since the settings from the puppet-server configuration should override the ones from puppet.conf at runtime in the context of a puppetserver master, I wasn't sure if this change would have any effect.  I guess it would if you were to try to run the puppet master under MRI Ruby directly, so I could see where this would be useful.